components/card/index.css

```diff
@@ -506,3 +507,9 @@
 		border-radius: 0;
 	}
 }
+
+@media (forced-colors: active) {
+	.spectrum-Card-coverPhoto {
+		border-block-end: var(--mod-card-border-width, var(--spectrum-card-border-width)) solid CanvasText;
```

Comment:
Maybe it's not necessary on that asset preview story. Adding that border does change the height of the cards, maybe because they're pulling in the `asset` component, so maybe it's more work outside of this branch. Feel free to disregard!

Reply:
I just happened to notice on the prod docs site that Asset (and default) look fine and show a border, because there is an `<hr>` Divider in the markup! That `<hr>` appears to be missing from our Storybook. I'm going to make a followup Jira issue for that and remove the high contrast change from this PR.